setup.py

```diff
@@ -79,16 +98,19 @@
     'jupyter_client',
     'ipython',
     'ipykernel', # bless IPython kernel for now
-    'prompt_toolkit>=1.0.0,<2.0.0',
+    'prompt_toolkit>=2.0.0,<2.1.0',
```

Comment:
Should we have an upper version number on prompt_toolkit? Is there something we know about 2.1.0?

Reply:
I basically just followed [ ipython/ipython#11177](https://github.com/ipython/ipython/pull/11177).
Maybe @takluyver (who made the `promt_toolkit 2` adoption for ipython), has some insigth on this.